app/admin/views/items.py

```diff
@@ -1,44 +1,40 @@
-from flask_admin.form import rules
+from flask_admin.form import BaseForm, rules
 from flask_admin.model.base import BaseView as FlaskBaseView
 from flask_sqlalchemy import BaseQuery
 from jinja2.runtime import Context
 from sqlalchemy.orm.scoping import scoped_session
+from typing import Type
 
 from .base import BaseView
 from ...models import Items, Storages, Users
 from ...utils.swagger_models import ItemsModels
-from ...utils.views import QuerySelectField
-
-
-def _get_user_label(user: Users) -> str:
-    label = f'{user.user_id}'
-    for value in [user.name, user.surname]:
-        if value is not None:
-            label += f' {value}'
-    return label
+from ...utils.views import get_user_label as _get_user_label, QuerySelectField
```

Comment:
We use `_` for protected/private fields/variables and methods/functions which should not be accessible from the outside. `get_user_label` is not like that.